files/usr/share/cinnamon/applets/window-list@cinnamon.org/applet.js

```diff
@@ -210,21 +224,26 @@ function AppMenuButton(applet, metaWindow, alert) {
 
 AppMenuButton.prototype = {
     _init: function(applet, metaWindow, alert) {
-        this.actor = new Cinnamon.GenericContainer({
-            name: 'appMenu',
-            style_class: 'window-list-item-box',
-            reactive: true,
-            can_focus: true,
-            track_hover: true });
+
+	this.actor = new Cinnamon.GenericContainer({
+	    name: 'appMenu',
+	    style_class: 'window-list-item-box',
+	    reactive: true,
+	    can_focus: true,
+	    track_hover: true });
 
         this._applet = applet;
         this.metaWindow = metaWindow;
         this.alert = alert;
 
         if (this._applet.orientation == St.Side.TOP)
             this.actor.add_style_class_name('window-list-item-box-top');
-        else
+        else if (this._applet.orientation == St.Side.BOTTOM)
             this.actor.add_style_class_name('window-list-item-box-bottom');
+        else if (this._applet.orientation == St.Side.LEFT)
+            this.actor.add_style_class_name('window-list-item-box-left');
+        else if (this._applet.orientation == St.Side.RIGHT)
+            this.actor.add_style_class_name('window-list-item-box-right');
```

Comment:
Completely agree.  I'd left it in line with what was there previously, but everytime I come back and look at this it jars horribly.  I was thinking of making the changes you suggest anyway, so I'll do that now.